terminal/TerminalServer.cpp

```diff
@@ -358,7 +427,130 @@ void startUserTerminal() {
   uth.run();
 }
 
-int main(int argc, char** argv) {
+void startJumpHostClient() {
+  string idpasskey = FLAGS_idpasskey;
+  if (FLAGS_idpasskeyfile.length() > 0) {
+    // Check for passkey file
+    std::ifstream t(FLAGS_idpasskeyfile.c_str());
+    std::stringstream buffer;
+    buffer << t.rdbuf();
+    idpasskey = buffer.str();
+    // Trim whitespace
+    idpasskey.erase(idpasskey.find_last_not_of(" \n\r\t") + 1);
+    // Delete the file with the passkey
+    remove(FLAGS_idpasskeyfile.c_str());
+  }
+  cout << "IDPASSKEY:" << idpasskey << endl;
```

Comment:
remove this cout

Reply:
Hey, this line was redirected to etclient for checking if ID & PASSKEY matches the one we have in the client side. It's similar to the cout logic in startUserTerminal. I feel we should keep it?

Reply:
Oh!  You are totally right, it actually breaks without that cout :-P.  Thanks for catching that.